oggm_3dviz/tools/viz.py

```diff
@@ -550,29 +550,78 @@ def update_glacier(self, step, camera_position_per_step=None):
         self.plotter.update()
 
     def export_animation(self, filename="animation.mp4", framerate=10,
-                         quality=5, moving_camera_start_and_end_point=None,
+                         quality=5, camera_trajectory='linear', linear_camera_start_and_end_point=None,
+                         rotate_camera_start_and_end_angle=None, rotate_camera_height=5, rotate_camera_radius=1,
                          **kwargs):
+        """
+        Export an animation of the glacier model.
+
+        Parameters:
+        - filename (str): The name of the output video file. Defaults to "animation.mp4".
+        - framerate (int): Frames per second for the video. Defaults to 10.
+        - quality (int): Quality of the output video (scale may vary based on the library). Defaults to 5.
+        - camera_trajectory (str): Type of camera movement ('linear' or 'rotate'). Defaults to 'linear'.
+        - linear_camera_start_and_end_point (tuple): Start and end points for the camera in 'linear' trajectory.
+            the start and end point is multiplied with the topography dimensions.
+            e.g. (0,0,0) is the topography center
+                 (1,0,0) is at the edge
+                 [(0, -1, 10),  (0, -0.5, 5)] zoom from an edge to the center
+        - rotate_camera_start_and_end_angle (tuple): Start and end angles for the camera in 'rotate' trajectory.
+            range from 0 to 360
+            e.g [200, 220]
+        - rotate_camera_height (int): The height of the rotated camera multiplied by elevation range. Defaults to 5.
+        - rotate_camera_radius (int): The radius of the rotated camera multiplied by map dimensions Defaults to 1.
+        - **kwargs: Additional keyword arguments to initialize the plotter.
+        """
+
+        # Initialize the plotter and glacier algorithm with additional parameters
         plotter, glacier_algo = self._init_plotter(**kwargs)
 
-        if moving_camera_start_and_end_point:
+        # Determine camera positions based on the chosen trajectory type
+        if camera_trajectory == 'linear':
```

Comment:
Instead of adding here code for differentiate between different camera trajectories it would be cleaner to do this in `utils`. So just keep here `camera_position_per_frame = get_camera_position_per_frame(self, camera_trajectory, kwarg_camera_trajectory)`, and implement all the different options in `get_camera_position_per_frame`. (maybe you can also move this out of `utils` and create a new file in oggm_3dviz/tools, something like moving_camera.py)